### ISSUE TITLE
Ability to pass aria-label when needed to the Link component

### ISSUE THREAD

**Issue by melniiv** [City-of-Helsinki/helsinki-design-system#808]
## To which feature the request relates to
https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/components/link/Link.tsx

## What
Add optional extraInformationAriaLabel property

## Why
When link is internal, and opened in the same tab, but the text is generic, for example "Read more", the aria-label should be still set for accessibility reasons.
